myapp/app/models/task.rb

```diff
@@ -5,11 +5,11 @@
 # Table name: tasks
 #
 #  id          :bigint           not null, primary key
-#  title       :string(255)
-#  description :string(255)
+#  description :string(255)      not null
+#  due_date    :datetime
 #  priority    :integer
 #  status      :integer
-#  due_date    :datetime
+#  title       :string(255)      not null
```

Comment:
auto created by gem `annotate` (sometimes it move around the order of columns that I cannot control 😅)